src/main/resources/conf/pottery.conf

```diff
@@ -0,0 +1,43 @@
+include "base-testnet.conf"
+
+mantis {
+  blockchains {
+    network = "pottery"
+  }
+
+  network {
+    protocol-version = 64
+
+    discovery {
+      discovery-enabled = true
```

Comment:
How does this work? Every node will need to be bootstrapped somewhere, right?

Reply:
As far as I know, every node uses bootstrapped nodes. I had some issues without discovery enabled. I can't recall what it was. : /